ckan-dev/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ckan-base:2.7
+
+MAINTAINER Open Knowledge International <info@okfn.org>
+
+ENV APP_DIR=/srv/app
+ENV SRC_EXTENSIONS_DIR=/srv/app/src_extensions
+
+# Install packages needed by the dev requirements
+RUN apk add --no-cache libffi-dev
+
+# Install CKAN dev requirements
+RUN pip install --no-binary :all: -r https://raw.githubusercontent.com/ckan/ckan/${GIT_BRANCH}/dev-requirements.txt
```

Comment:
Do we need to pull dev-requirements.txt from github? CKAN is installed in the base image, so we can make a reference to the local one?